### PR TITLE
Add info about new entities for easyEnergy

### PR DESCRIPTION
**Pull request by klaasnicolaas**
## Proposed change

Adds information about 2 new entities for easyEnergy.
The text could possibly be better about explaining how to interpret the data.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/90543
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/easyenergy.markdown [hunk 1]
```diff
@@ -47,6 +47,18 @@ prices for electricity that you use (buy) or return (sell). Every day around
...
+- Number of hours with the current price higher or lower
+
+Entities with the number of hours, indicates how many hours there are with a price
+**above** or **below** the current hourly price. If we take the graph below as an example
+and it is 00:30, then there are 8 hours below the current price and 4 hours above the
+current price. With this information you could switch a device at the X cheapest number
+of hours during the day.
+
+<p class='img'>
+  <img src='/images/integrations/easyenergy/pricegraph.png' alt='Screenshot showing energy price graph.'>
```
Comment (frenck) [inline source/_integrations/easyenergy.markdown:59]: The graph shown isn't possible to make with the built-in provided graphing. Adding graphs like these, make users think, however, it is possible. I would not recommend it for that reason.

File: source/_integrations/easyenergy.markdown [hunk 2]
```diff
@@ -47,6 +47,18 @@ prices for electricity that you use (buy) or return (sell). Every day around
 - Time of day when the price is highest
 - Time of day when the price is at its lowest
 - Percentage of the current price compared to the maximum price
+- Number of hours with the current price higher or lower
+
+Entities with the number of hours, indicates how many hours there are with a price
+**above** or **below** the current hourly price. If we take the graph below as an example
+and it is 00:30, then there are 8 hours below the current price and 4 hours above the
+current price. With this information you could switch a device at the X cheapest number
+of hours during the day.
```
Comment (frenck) [inline source/_integrations/easyenergy.markdown]:
```suggestion
Entities with the number of hours indicate how many hours there are with a price
**above** or **below** the current hourly price. If we take the graph below as an example
and it is 00:30, then there are 8 hours below the current price and 4 hours above the
current price. With this information, you could switch devices at the X cheapest number
of hours during the day.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/easyenergy.markdown (hunk 1 above)
Comment (klaasnicolaas) [reply source/_integrations/easyenergy.markdown:59]: It's a graph from [easyEnergy](https://www.easyenergy.com/nl/energietarieven) that I made a screenshot of. I also added it not with the intention that users can create this, but as a visual image to better understand how the new entities work 🤷🏻‍♂️ 

It also took me some time to understand how exactly the new feature should be understood. That is why I would like to add it as clearly as possible in the documentation.

Comment (frenck) [reply source/_integrations/easyenergy.markdown:59]: That I didn't realize, yeah clarifying might help there 👍

**Review by frenck — APPROVED** [review]

**Review by frenck — APPROVED** [review]

**Pull request merged**